app/workers/users/self_delete_worker.rb

```diff
@@ -1,6 +1,8 @@
 module Users
-  class SelfDeleteJob < ApplicationJob
-    queue_as :users_self_delete
+  class SelfDeleteWorker
+    include Sidekiq::Worker
+
+    sidekiq_options queue: :high_priority, retry: 10
 
     def perform(user_id, service = Users::Delete)
```

Comment:
Since we are not using this service argument can you remove it and explicitly call the service below?
```ruby
Users::Delete.call(user)
```

Reply:
sure, i will